app/partials/bc-logo-footer.pug

```diff
@@ -0,0 +1,16 @@
+.flex-row.flex-center.flex-column-mobile.flex-between.bc-logo-footer.width-100.width-100-tablet
+  .container
+    .flex-row.flex-center.flex-between.flex-column-mobile
+      .flex-row.flex-center.flex-column-mobile.em-300
+        img(src="img/blue-logo.svg" alt="" height="40")
+        a.upper.white.ml-40(href="{{rootURL}}") .info Explorer
+        a.upper.white.ml-40(href="https://www.blockchain.com/about") About
+        a.upper.white.ml-40(href="https://blog.blockchain.com") Blog
+        a.upper.white.ml-40(href="https://support.blockchain.com") Support
+      li.dropup(uib-dropdown)
```

Comment:
This dropdown gets cut off by the header at smaller browser heights, could you fix that?